src/DssSpell.t.sol

```diff
@@ -1111,4 +1120,51 @@ contract DssSpellTest is DSTest, DSMath {
         assertEq(MedianAbstract(UNIUSD_MED).bud(SET_UNI), 1);
     }
 
+    event Log(string message, address deployer, bytes32 contractName);
+
+    function hasWards(address addr) internal returns (bool) {
+        (bool ok, bytes memory data) = addr
+            .call(abi.encodeWithSignature("wards(address)", 0x0));
+        return ok && data.length == 32;
+    }
+
+    address[] evilAddresses = [
```

Comment:
I'd just call them `deployerAddresses`

Reply:
Fair enough. I was just thinking about the possibility to expand this in the future to other addresses we want to check that aren't deployers. But I can't really think about good examples of what these might be.

Reply:
Not evil, just misunderstood.